server/routes/viewModels/taskLists/caTasks.js

```diff
@@ -147,7 +145,12 @@ module.exports = {
       return [proposedAddressTask, curfewAddressTask, bassTask, refusalTask].filter(task => task.visible)
     }
 
+    if (!eligible) {
+      return [eligibilityTask, informOffenderTask]
+    }
+
     return [
+      eligibilityTask,
```

Comment:
no need for informOffenderTask here?

Reply:
No, because that task only appears when the offender is Not eligible and that particular line cannot be reached when that is the case.